src/daemon/XorgDisplayServer.cpp

```diff
@@ -128,6 +129,12 @@ namespace SDDM {
 
         if (daemonApp->testing()) {
             QStringList args;
+            QDir x11socketDir(QStringLiteral("/tmp/.X11-unix"));
+            int display = 100;
```

Comment:
I used 100 to underline the fact, that this is just a virtual display. Also, if there is some user script which opens a new display with fixed number while the test mode is on, it could find this display number be blocked and break. I think it should do no harm?

Thanks for having a look into this!